src/pages/auth/Signup.jsx

```diff
@@ -0,0 +1,65 @@
+import { Link } from "react-router-dom";
+import "./auth.css";
+import {useState} from "react";
+
+export function Signup(){
+    const [visibility, setVisibility] = useState(false);
+    const [visibility1, setVisibility1] = useState(false);
```

Comment:
suggestion: naming, eg: isPasswordVisible, isConfPassWrodVisible

Reply:
Thank you for your suggestion, I have updated it.